source/isaaclab/isaaclab/terrains/terrain_importer.py

```diff
@@ -176,12 +176,13 @@ def set_debug_vis(self, debug_vis: bool) -> bool:
     Operations - Import.
     """
 
-    def import_ground_plane(self, key: str, size: tuple[float, float] = (2.0e6, 2.0e6)):
+    def import_ground_plane(self, key: str, size: tuple[float, float] = (2.0e6, 2.0e6), color: tuple[float, float, float] = (0.0, 0.0, 0.0)):
         """Add a plane to the terrain importer.
 
         Args:
             key: The key to store the mesh.
             size: The size of the plane. Defaults to (2.0e6, 2.0e6).
+            color: The color of the plane. Defaults to (0.0, 0.0, 0.0).
```

Comment:
Probably a reminant of previous code, but why do we pass in `size` and `color` if they are getting propogated through self.cfg? I don't think that's necessary?